src/simba/class_session/detail/includes/edit_review_modal.html

```
@@ -0,0 +1,166 @@
+                class="peer -ms-7 size-7 bg-transparent border-0 text-transparent cursor-pointer appearance-none checked:bg-none focus:bg-none focus:ring-0 focus:ring-offset-0"
+                name="rating" value="5" checked />
+              <label for="rating"
+                class="peer-checked:text-yellow-500 peer-hover:text-yellow-500 text-stone-300 pointer-events-none dark:peer-checked:text-yellow-400 dark:peer-hover:text-yellow-400 dark:text-neutral-600">
+                <svg class="shrink-0 size-7" xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+                  viewBox="0 0 24 24" fill="currentColor" stroke="none" stroke-width="1.5" stroke-linecap="round"
+                  stroke-linejoin="round">
+                  <polygon
+                    points="12 2 15.09 8.26 22 9.27 17 14.14 18.18 21.02 12 17.77 5.82 21.02 7 14.14 2 9.27 8.91 8.26 12 2" />
+                </svg>
+              </label>
+              <input id="rating" type="radio"
```

HTMLHint (1.5.0)

[error] The id value [ rating ] must be unique.

(id-unique)